proposed_solutions/core.clj

```diff
@@ -0,0 +1,77 @@
+;from core.clj by Larissa Gaulia
+
+(ns cipher.core)
+
+(defn abs
+  [x y]
+  (if (< x y) (- y x) (- x y)))
+
+
+(defn to-int
+  "takes a lowercase letter character and returns its position in the alphabet: a = 0, b = 1, etc."
+  [letter-char]
+  (let [ascii-a (int \a)]
+    (- (int letter-char) ascii-a)))
+
+(defn to-char
+  "takes a position in the alphabet and returns its lowercase char"
+  [position]
+  (let [int-a (int \a)]
+    (char (+ (int position) int-a))))
+
+(defn shift
+  "makes the caeser cipher"
+  [letter shifter]
+  (to-char (mod(+ (to-int letter) shifter) 26)))
+
+(defn caesar-encrypt
+  "encrypting a word w with a key k using Caesar cipher"
+  [w k]
+  (apply str (mapv #(shift % k) w)))
+
+(defn caesar-decrypt
+  "decrypting a word w with a key k using Caesar cipher"
+  [w k]
+  (apply str (mapv #(shift % (- 0 k)) w)))
```

Comment:
Minor, you can instead `(- 0 k)` -> `(- k)`

Reply:
simpler and easier. tks! 👍